Makefile.PL

```diff
@@ -178,21 +178,13 @@ sub MY::processPL
     } elsif ( $rule =~ m{^\s*^RealPPPort\.xs\s+:}m ) {
       $rule =~ s{^(\s*^RealPPPort\.xs\s+:.*)}{$1 ppport.h}m; # RealPPPort.xs depends on ppport.h
       $rule =~ s{pm_to_blib}{}m; # RealPPPort.xs is used to build PPPort binary, so it cannot depend on it
-    } elsif ( $rule =~ m{^\s*^t/01_test\.t\s+:}m ) {
-      $rule =~ s{^(\s*^t/01_test\.t\s+:.*)}{$1 $includes}m; # all tests in t/ depends on all files from parts/inc
+    } elsif ( $rule =~ m{^\s*\S+\b01_test\.t\s+:}m ) {
+      $rule =~ s{^(\s*^\S+\b01_test\.t\s+:.*)}{$1 $includes}m; # all tests in t/ depends on all files from parts/inc
       $rule =~ s{pm_to_blib}{}m; # Generating test files does not depend on built PPPort in blib/
     }
     $updated .= $rule;
   }
 
-  # All test targets depends on pure_all target, so ensure that t/01_test.t generated by mktests.PL is up-to-date
-  $updated .= <<'TESTS_IN_T';
-
-pure_all :: t/01_test.t
-	$(NOECHO) $(NOOP)
```

Comment:
Do not remove this code, see above comment. Without this Makefile target you are breaking running `make test`.

Reply:
It's definitely redundant in blead because MakeMaker has already generated the same rule, but perhaps blead has a later version of MakeMaker than what you're using.  Duplicating the rule should not do any real harm that I can think of, but if it's left in, the dependency must be constructed with `catfile('t', '01_test.t')` so that it appears in the native syntax of whatever platform it's running on.

Reply:
Devel::PPPort is for older perl versions, not (only) for blead. So you cannot expect that MakeMaker is in very new version which has fixed all bugs.

Reply:
`catfile` is of course correct function call.